CMakeLists.txt

```
@@ -750,6 +750,11 @@ endif()
 
 add_custom_target(extra_test ${CMAKE_CTEST_COMMAND} -C ExtraTests)
 
+# must activate avx2 flags for mixed precision.
+if(HAVE_AVX2_EXTENSION)
+  set_property(SOURCE ${AVX2_SOURCE_FILES} PROPERTY COMPILE_FLAGS ${AVX2_FLAGS})
```

To get the correct CFLAGS please do:
```suggestion
  set_property(SOURCE ${AVX2_SOURCE_FILES} PROPERTY COMPILE_FLAGS ${AVX2_FLAGS})
  set(CMAKE_C_STANDARD C11)
```
With that compilation succeeds for me without manually setting CMAKE_C_FLAGS="-O3 -std=c11 -D_ISOC11_SOURCE -march=native"